deps.windows/60-qrcodegencpp.ps1

```diff
@@ -0,0 +1,111 @@
+param(
+    [string] $Name = 'qrcodegencpp',
+    [string] $Version = '1.8.0',
+    [string] $Uri = 'https://github.com/nayuki/QR-Code-generator.git',
+    [string] $Hash = '720f62bddb7226106071d4728c292cb1df519ceb',
+    [string] $UriCMake = 'https://github.com/EasyCoding/qrcodegen-cmake.git',
+    [string] $HashCMake = 'c57623d48a2d422b0f908dcf22d56d603e56c0e4'
+)
+
+function Setup {
+    Setup-Dependency -Uri $Uri -Hash $Hash -DestinationPath "$Path/src"
+    Setup-Dependency -Uri $UriCMake -Hash $HashCMake -DestinationPath "$Path/qrcodegen-cmake"
+}
+
+function Clean {
+    Set-Location "$Path/src"
+
+    if ( Test-Path "build_${Target}" ) {
+        Log-Information "Clean build directory (${Target})"
+        Remove-Item -Path "build_${Target}" -Recurse -Force
+    }
+}
+
+function Patch {
+    Log-Information "Patch (${Target})"
+    Set-Location $Path
+
+    $Items = @(
+        @{
+            Path = "qrcodegen-cmake/CMakeLists.txt"
+            Destination = "src"
+            ErrorAction = 'SilentlyContinue'    
+        }
+        @{
+            Path = "qrcodegen-cmake/cmake"
+            Destination = "src"
+            Recurse = $true
+            ErrorAction = 'SilentlyContinue'    
+        }
+    )
+
+    $Items | ForEach-Object {
+        $Item = $_
+        Copy-Item @Item
+    }
+}
+
+function Configure {
+    Log-Information "Configure (${Target})"
+    Set-Location "$Path/src"
+
+	$OnOff = @('OFF', 'ON')
+    $Options = @(
+        $CmakeOptions
+        "-DBUILD_SHARED_LIBS:BOOL=$($OnOff[$script:Shared.isPresent])"
+    )
+
+    Invoke-External cmake -S . -B "build_${Target}" @Options
+}
+
+function Build {
+    Log-Information "Build (${Target})"
+    Set-Location "$Path/src"
+
+    $Options = @(
+        '--build', "build_${Target}"
+        '--config', $Configuration
+    )
+
+    if ( $VerbosePreference -eq 'Continue' ) {
+        $Options += '--verbose'
+    }
+
+    Invoke-External cmake @Options
+}
+
+function Install {
+    Log-Information "Install (${Target})"
+    Set-Location "$Path/src"
+
+    $Options = @(
+        '--install', "build_${Target}"
+        '--config', $Configuration
+    )
+
+    if ( $Configuration -match "(Release|MinSizeRel)" ) {
+        $Options += '--strip'
+    }
+
+    Invoke-External cmake @Options
+}
+
+function Fixup {
+    Log-Information "Fixup (${Target})"
+    Set-Location $Path
+
+    $Items = @(
+        @{ FullPath = "$($ConfigData.OutputPath)/include/qrcodegen" }
+        @{ FullPath = "$($ConfigData.OutputPath)/lib/cmake/qrcodegen" }
+        @{ FullPath = "$($ConfigData.OutputPath)/lib/pkgconfig/qrcodegen.pc" }
+        @{ FullPath = "$($ConfigData.OutputPath)/lib/qrcodegen.*" }
+        @{ FullPath = "$($ConfigData.OutputPath)/bin/qrcodegen.*" }
+    )
+
+    $Items | ForEach-Object {
+    $Item = $_
```

Comment:
Needs indentation.

Reply:
Sorry, I only see this review weeks later.

Fixed.